manifests/h/Henry++/MemReduct/3.3.5/Henry++.MemReduct.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Henry++.MemReduct
+PackageVersion: 3.3.5
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Platform:
+  - Windows.Desktop
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/henrypp/memreduct/releases/download/v.3.3.5/memreduct-3.3.5-setup.exe
+  InstallerSha256: 3555B6148F8BF9415C1B0DB8B03C649B530670C7775631D6F26FDA5EEE547F24
+  UpgradeBehavior: install
+  InstallerSwitches:
+    Custom: /S
```

Comment:
Custom is meant for other switches then the silent switch :)

Reply:
Yes, I refer to the `chocolatey` package for this custom switch.
And `Henry++` doesn't seem to set an `ID` for his software.